src/main/app/intercom.test.js

```diff
@@ -65,13 +63,12 @@ describe('main/app/intercom', () => {
             ModalManager.addModal.mockReturnValue(promise);
 
             handleWelcomeScreenModal();
-            expect(ModalManager.addModal).toHaveBeenCalledWith('welcomeScreen', '/some/index.html', '/some/preload.js', null, {}, true);
+            expect(ModalManager.addModal).toHaveBeenCalledWith('welcomeScreen', 'mattermost-desktop://renderer/welcomeScreen.html', '/some/preload.js', null, {}, true);
```

Comment:
Worth making the string `mattermost-desktop://renderer` an exported constant as its being used in a lot of places and making it a constant will avoid possible typos.

Reply:
I'm actually thinking I might just do a helper method like I had before with `getLocalURLString`, just so I can also control how things are concatenated. Good call nonetheless :)